controller/source/kv_client/rocksdb.hpp

```diff
@@ -38,7 +38,7 @@ class rocksdb_client : public kv_client
       // std::cout << "GET operation succeeded! Key: " << key << ", Value: " << response.get_data() << std::endl;
       return response.get_data();
     }
-    std::cout << "GET operation failed" << std::endl;
+    // std::cout << "GET operation failed" << std::endl;
```

Comment:
I mostly left the outputs to make sure that the workloads actually succeed. However, it's time to remove the prints. 
I'll do manual test to verify everything works as it should.